src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy

```diff
@@ -116,6 +116,10 @@ class SwitchHelper {
         sw.nbFormat().description
     }
 
+    static String getHwSwString(Switch sw) {
```

Comment:
`@Memoized`?

Reply:
Not a big deal, since it already calls a Memoized 'nbFormat' method, but added here as well